lesson18/index.js

```diff
@@ -88,6 +109,59 @@ const controlBtnBehavior = () => {
   nextBtn.classList.remove("is-disabled");
 }
 
+const setPagenation = (slides) => {
+  const slider = document.getElementById("js-slider");
+  const pagenation = document.createElement("div");
+  pagenation.classList.add("pagenation");
+  const pagenationWrapper = document.createElement("div");
+  pagenationWrapper.classList.add("pagenationWrapper");
+  const slideLength = slides.length;
+
+  slides.map((slide,index) => {
```

Comment:
IMO
map is a method that returns an array.
It does not seem appropriate to use map when the return value is not used.
How about using a for statement.
```suggestion
  for (let i = 0; i < slideLength ; i++) {
```

Reply:
[fe0a6d3](https://github.com/hippopo-moo/js-handson-work/pull/20/commits/fe0a6d374aa36b75521b21f847c7a76efc4fa05e)

I wanted to use index, so I used map.
I guess the for statement would have been sufficient since I was getting the activateBulletIndex separately in the function.
Thanks, I fixed it!